Documentation/ceph-object-multisite.md

```diff
@@ -18,7 +19,7 @@ When a ceph-object-store is created with the `zone` section, the ceph-object-sto
 
 This allows the ceph-object-store to replicate its data over multiple Ceph clusters.
 
-To review core multisite concepts please read the [ceph-multisite design overview](/design/ceph/object/ceph-multisite-overview.md).
+To review core multisite concepts please read the [ceph-multisite design overview](https://github.com/rook/rook/blob/{{ branchName }}/design/ceph/object/ceph-multisite-overview.md).
```

Comment:
You'll need to put the link all on a single line, I believe the doc generation will be broken if this var is on multiple lines.